src/master-playlist-controller.js

```diff
@@ -176,40 +56,45 @@ const parseCodecs = function(codecs) {
 };
 
 export default class MasterPlaylistController extends videojs.EventTarget {
-  constructor({url, withCredentials, currentTimeFunc, mediaSourceMode, hlsHandler,
-    externHls}) {
+  constructor({
+    url,
+    withCredentials,
+    mode,
+    tech,
+    bandwidth,
+
```

Comment:
Not really. I was using it as a separator for "dependencies that need to be removed" :)